<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timewidget</class>
 <widget class="QWidget" name="timewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_year">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>136</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>136</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>136</width>
     <height>62</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Microsoft YaHei UI&quot;;
border-color: rgba(151, 151, 151, 1);
background-color: rgb(0, 0, 0);
color: rgba(13, 53, 136, 1);
border: 1px solid #979797;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: MicrosoftYaHei;
font-size: 32px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_month">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>86</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Microsoft YaHei UI&quot;;
border-color: rgba(151, 151, 151, 1);
background-color: rgb(0, 0, 0);
color: rgba(13, 53, 136, 1);
border: 1px solid #979797;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_day">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>86</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Microsoft YaHei UI&quot;;
border-color: rgba(151, 151, 151, 1);
background-color: rgb(0, 0, 0);
color: rgba(13, 53, 136, 1);
border: 1px solid #979797;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: MicrosoftYaHei;
font-size: 32px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: MicrosoftYaHei;
font-size: 32px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_minute">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>86</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Microsoft YaHei UI&quot;;
border-color: rgba(151, 151, 151, 1);
background-color: rgb(0, 0, 0);
color: rgba(13, 53, 136, 1);
border: 1px solid #979797;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_hour">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>86</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>62</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40px &quot;Microsoft YaHei UI&quot;;
border-color: rgba(151, 151, 151, 1);
background-color: rgb(0, 0, 0);
color: rgba(13, 53, 136, 1);
border: 1px solid #979797;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: MicrosoftYaHei;
font-size: 32px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>32</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: MicrosoftYaHei;
font-size: 32px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#close{

background-image:url(:/29.png);
border:none;
font: 75 14pt &quot;Microsoft YaHei UI&quot;;
	color: rgb(255, 255, 255);
}
QPushButton#close:pressed{
background-image: url(:/侧键_关闭弹窗_按下.png);
border:none;
font: 75 14pt &quot;Microsoft YaHei UI&quot;;
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
